src/components/structures/login/Login.js

```diff
@@ -223,14 +224,14 @@ module.exports = React.createClass({
                  !this.state.enteredHomeserverUrl.startsWith("http")))
             {
                 errorText = <span>
-                    Can't connect to homeserver via HTTP when an HTTPS URL is in your browser bar.
-                    Either use HTTPS or <a href='https://www.google.com/search?&q=enable%20unsafe%20scripts'>enable unsafe scripts</a>
+                    { counterpart.translate("Can't connect to homeserver via HTTP when an HTTPS URL is in your browser bar")}.
+                    { counterpart.translate('Either use HTTPS or ')}<a href='https://www.google.com/search?&q=enable%20unsafe%20scripts'>{ counterpart.translate('enable unsafe scripts')}</a>
```

Comment:
@krombel hm not sure about it as google should autodetect the browser language. and riot does the requests via the client so that should work for itself.